### PR TITLE
added query parsing and tag list detection for the tag input field

### PR DESCRIPTION
**Pull request by Pomax**
Related issue: https://github.com/mozilla/network-pulse/issues/1011
Summary: adds support for presupplied tags using `add/?tags=a,b,c,etc`

### CODE REVIEW

File: js/utility.js [hunk 1]
```diff
@@ -26,6 +26,22 @@ const Utils = {
     return console.error(`No window available to get the current url!`);
   },
 
+  getCurrentURLQuery() {
```
Comment (gideonthomas) [inline js/utility.js]: any reason we don't just use https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams?

File: pages/add/form/auto-complete-input.js [hunk 2]
```diff
@@ -30,6 +31,24 @@ export default class AutoCompleteInput extends Component {
     };
   }
 
+  /**
```
Comment (gideonthomas) [inline pages/add/form/auto-complete-input.js]: probably needs to be in this file instead: https://github.com/mozilla/network-pulse/blob/master/pages/add/form/tags.js#L4

File: js/utility.js (hunk 1 above)
Comment (Pomax) [reply js/utility.js]: mostly that when last I checked, that wasn't broadly supported, and I never bothered to check if that had changed in the mean time! Good call =)

**Comment by Pomax** [comment]
tested on https://network-pulse-staging-pr-1015.herokuapp.com/entry/462?tags=prefilled%2Ctags by hitting up https://network-pulse-staging-pr-1015.herokuapp.com/add?tags=prefilled,tags and then saving with only a title and url, no editing of the tags beyond what the autoloading does.

**Pull request merged**